### PR TITLE
Streamlined for 6.4

### PR DESCRIPTION
**Pull request by npoyant**
Took the latest merge  and cleaned up variables, syntax issues, and overall presentation of the variables to alleviate confusion.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,43 +1,48 @@
...
+  "foreman-initial-organization": "{{ satellite_deployment_organization }}"
+  "foreman-initial-location": "{{ satellite_deployment_location }}"
+  "foreman-admin-username": "{{ satellite_deployment_admin_username }}"
+  "foreman-admin-password": "{{ satellite_deployment_admin_password }}"
 
 
 satellite_deployment_ipa_authentication: "false"
 
 # configure_satellite:
+<<<<<<< HEAD
```
Comment (itewk) [inline defaults/main.yml]: @npoyant merge issue

File: defaults/main.yml [hunk 2]
```diff
@@ -49,6 +54,21 @@ satellite_deployment_repositories:
 # configure_capsules:
 satellite_deployment_puppet_env: "production"
 
+<<<<<<< HEAD
```
Comment (itewk) [inline defaults/main.yml]: @npoyant merge issue

File: defaults/main.yml [hunk 3]
```diff
@@ -91,3 +111,41 @@ satellite_deployment_fw_ports:
 
 satellite_deployment_installer_cmd: "/usr/sbin/satellite-installer
   --scenario satellite"
+<<<<<<< HEAD
```
Comment (itewk) [inline defaults/main.yml]: @npoyant merge issue

File: tasks/install_satellite.yml [hunk 4]
```diff
@@ -87,11 +81,18 @@
     satellite_deployment_installer_cmd: "{{ satellite_deployment_installer_cmd
       }} --{{ item.key }}='{{ item.value }}'"
   with_dict: "{{ satellite_deployment_answers }}"
+<<<<<<< HEAD
```
Comment (itewk) [inline tasks/install_satellite.yml]: @npoyant merge issue

File: tasks/install_satellite.yml [hunk 5]
```diff
@@ -103,11 +104,16 @@
   - "update_satellite"
     
 #Run the install of Satellite 6.4
+>>>>>>> f4b18ade8ef9e1075a8c5b60015840ca170310de
 - name: "Satellite installation"
   command: "{{ satellite_deployment_installer_cmd }}"
   become: "yes"
   when: foreman_service_file.stat.exists == True and
+<<<<<<< HEAD
```
Comment (itewk) [inline tasks/install_satellite.yml]: @npoyant merge issue

File: tasks/main.yml [hunk 6]
```diff
@@ -1,11 +1,18 @@
 ---
 - include: set_facts.yml
 - include: registration.yml
+<<<<<<< HEAD
```
Comment (itewk) [inline tasks/main.yml:4]: @npoyant merge issue

File: tasks/main.yml [hunk 7]
```diff
@@ -1,11 +1,18 @@
...
+  when: satellite_connected == "True"
+=======
 - include: network.yml
   when: satellite_deployment_set_network
+>>>>>>> f4b18ade8ef9e1075a8c5b60015840ca170310de
 - include: firewall.yml
 - include: install_software.yml
 - include: install_satellite.yml
 - include: configure_satellite.yml
+<<<<<<< HEAD
```
Comment (itewk) [inline tasks/main.yml:14]: @npoyant merge issue

File: templates/answerfile.katello-installer.yaml.j2 [hunk 8]
```diff
@@ -23,7 +23,11 @@
     admin_password: "{{ satellite_deployment_admin_password }}"
     initial_organization: "{{ satellite_deployment_organization }}"
     initial_location: "{{ satellite_deployment_location }}"
+<<<<<<< HEAD
```
Comment (itewk) [inline templates/answerfile.katello-installer.yaml.j2:26]: @npoyant merge issue

File: templates/answerfile.katello-installer_6.4.yaml.j2 [hunk 9]
```diff
@@ -12,6 +12,22 @@ katello:
 foreman:
   organizations_enabled: true
   locations_enabled: true
+<<<<<<< HEAD
```
Comment (itewk) [inline templates/answerfile.katello-installer_6.4.yaml.j2:15]: @npoyant merge issue

File: templates/answerfile.katello-installer_6.4.yaml.j2 [hunk 10]
```diff
@@ -29,12 +45,32 @@ foreman:
...
   passenger_ruby: "/usr/bin/tfm-ruby"
   passenger_ruby_package: tfm-rubygem-passenger-native
+>>>>>>> f4b18ade8ef9e1075a8c5b60015840ca170310de
   keepalive: true
   max_keepalive_requests: 10000
   admin_username: "{{ satellite_deployment_admin_username }}"
   admin_password: "{{ satellite_deployment_admin_password }}"
   initial_organization: "{{ satellite_deployment_organization }}"
   initial_location: "{{ satellite_deployment_location }}"
+<<<<<<< HEAD
```
Comment (itewk) [inline templates/answerfile.katello-installer_6.4.yaml.j2:55]: @npoyant merge issue

File: templates/answerfile.katello-installer_6.4.yaml.j2 [hunk 11]
```diff
@@ -55,12 +91,32 @@ foreman_proxy:
...
   ssl_port: '9090'
   templates: false
+>>>>>>> f4b18ade8ef9e1075a8c5b60015840ca170310de
   ssl_ca: "/etc/foreman-proxy/ssl_ca.pem"
   ssl_cert: "/etc/foreman-proxy/ssl_cert.pem"
   ssl_key: "/etc/foreman-proxy/ssl_key.pem"
   foreman_ssl_ca: "/etc/foreman-proxy/foreman_ssl_ca.pem"
   foreman_ssl_cert: "/etc/foreman-proxy/foreman_ssl_cert.pem"
   foreman_ssl_key: "/etc/foreman-proxy/foreman_ssl_key.pem"
+<<<<<<< HEAD
```
Comment (itewk) [inline templates/answerfile.katello-installer_6.4.yaml.j2:101]: @npoyant merge issue

File: templates/answerfile.katello-installer_6.4.yaml.j2 [hunk 12]
```diff
@@ -87,12 +143,29 @@ foreman_proxy::plugin::openscap:
...
   enabled: true
   pulpnode_enabled: false
+>>>>>>> f4b18ade8ef9e1075a8c5b60015840ca170310de
 foreman_proxy::plugin::discovery: true
 foreman_proxy_content:
   pulp_master: true
   templates: false
   qpid_router_broker_addr: localhost
   enable_deb: false
+<<<<<<< HEAD
```
Comment (itewk) [inline templates/answerfile.katello-installer_6.4.yaml.j2:153]: @npoyant merge issue

**Review by itewk — CHANGES_REQUESTED** [review]

**Comment by npoyant** [comment]
closed to resolve merge issues

**Pull request closed without merging**